swagger.yaml

```diff
@@ -4532,6 +4532,8 @@ definitions:
         type: integer
         minimum: 1
         maximum: 32
+      inactive_deletion_hours:
+        type: number
```

Comment:
```suggestion
        type: integer
```

Reply:
time.Duration.Hours() function return `float64`, this is why I have chosen `number` instead of `integer` - otherwise we need an additional cast.

Reply:
Also don't we want to allow floating points? let's say `10.5` hours? why should we restrict to integer?

Reply:
Changed it to `integer`.